python/src/MetropolisHastingsImplementation_doc.i.in

```diff
@@ -0,0 +1,499 @@
+%define OT_MetropolisHastings_doc
+"Base class for Metropolis-Hastings algorithms.
+
+Available constructor:
+    MetropolisHastingsImplementation(*targetDistribution, initialState, marginalIndices*)
```

Comment:
There seem to be some missing inputs in the constructor: proposal distribution, range

Reply:
the instrumental distribution is part of the RWMH ctor, and range is renamed support